src/test/java/org/sagebionetworks/bridge/exporter/integration/ExportTest.java

```diff
@@ -0,0 +1,813 @@
+    private Project project;
+    private Team team;
+
+    private String env;
+    private String recordIdOverrideBucket;
+    private String s3FileName;
+
+    private Table ddbExportTimeTable;
+    private Table ddbSynapseMetaTables;
+    private Table ddbSynapseTables;
+    private Table ddbRecordTable;
+
+    private SqsHelper sqsHelper;
+    private S3Helper s3Helper;
+    private AmazonS3Client s3Client;
+    private DynamoScanHelper ddbScanHelper;
+
+    @Autowired
+    public final void setBridgeConfig(Config bridgeConfig) {
+        this.bridgeConfig = bridgeConfig;
+    }
+
+    @Autowired
+    public final void setEnv(String env) {
+        this.env = env;
+    }
+
+    @Autowired
+    public final void setSqsHelper(SqsHelper sqsHelper) {
+        this.sqsHelper = sqsHelper;
+    }
+
+    @Autowired
+    public final void setS3Helper(S3Helper s3Helper) {
+        this.s3Helper = s3Helper;
+    }
+
+    @Autowired
+    public final void setS3Client(AmazonS3Client s3Client) {
+        this.s3Client = s3Client;
+    }
+
+    @Autowired
+    final void setDdbScanHelper(DynamoScanHelper ddbScanHelper) {
+        this.ddbScanHelper = ddbScanHelper;
+    }
+
+    @Resource(name = "ddbExportTimeTable")
+    final void setDdbExportTimeTable(Table ddbExportTimeTable) {
+        this.ddbExportTimeTable = ddbExportTimeTable;
+    }
+
+    @Resource(name = "ddbSynapseMetaTables")
+    final void setDdbSynapseMetaTables(Table ddbExportTimeTable) {
+        this.ddbSynapseMetaTables = ddbExportTimeTable;
+    }
+
+    @Resource(name = "ddbSynapseTables")
+    final void setDdbSynapseTables(Table ddbExportTimeTable) {
+        this.ddbSynapseTables = ddbExportTimeTable;
+    }
+
+    @Resource(name = "ddbRecordTable")
+    public final void setDdbRecordTable(Table ddbRecordTable) {
+        this.ddbRecordTable = ddbRecordTable;
+    }
+
+    private void setupProperties() throws IOException {
+        SYNAPSE_USER = bridgeConfig.get(USER_NAME);
+        SYNAPSE_API_KEY = bridgeConfig.get(SYNAPSE_API_KEY_NAME);
+        EXPORTER_SYNAPSE_USER_ID = bridgeConfig.get(EXPORTER_SYNAPSE_USER_ID_NAME);
+        TEST_USER_ID = Long.parseLong(bridgeConfig.get(TEST_USER_ID_NAME));
+        EXPORTER_SQS_URL = bridgeConfig.get("exporter.request.sqs.queue.url");
+        recordIdOverrideBucket = bridgeConfig.get(BridgeExporterUtil.CONFIG_KEY_RECORD_ID_OVERRIDE_BUCKET);
+    }
+
+    @Before
+    public void before() throws Exception {
+        setupProperties();
+
+        admin = TestUserHelper.getSignedInAdmin();
+        synapseClient = new SynapseAdminClientImpl();
+        synapseClient.setUserName(SYNAPSE_USER);
+        synapseClient.setApiKey(SYNAPSE_API_KEY);
+
+        // create test study
+        StudiesApi studiesApi = admin.getClient(StudiesApi.class);
+
+        studyId = Tests.randomIdentifier(ExportTest.class);
+        Study study = Tests.getStudy(studyId, null);
+        studiesApi.createStudy(study).execute().body();
```

Comment:
i think creating separate study for each test is better since we need to verify what's being exported to synapse table for each test, its better to leave each study and synapse table 'clean' before each test.